src/states_screens/race_gui.hpp

```diff
@@ -120,6 +130,9 @@ class RaceGUI : public RaceGUIBase
     /** Display items that are shown once only (for all karts). */
     void drawGlobalMiniMap     ();
     void drawGlobalTimer       ();
+    //TODO : support live difference computation outside of ghost replays
```

Comment:
Is this TODO really useful? We needen't specify all possible future eventual improvements as todos all over the code